client/settings/fraud-protection/advanced-settings/index.tsx

```diff
@@ -69,37 +63,38 @@ const observerEventMapping: Record< string, string > = {
 		'wcpay_fraud_protection_advanced_settings_card_items_threshold_viewed',
 };
 
-const Breadcrumb = () => (
+const AdvancedFraudSettingsDescription = () => (
 	<>
-		<h2 className="fraud-protection-header-breadcrumb">
-			<Link
-				type="wp-admin"
-				href={ getAdminUrl( {
-					page: 'wc-settings',
-					tab: 'checkout',
-					section: 'woocommerce_payments',
-				} ) }
-			>
-				{ 'WooPayments' }
-			</Link>
-			&nbsp;&gt;&nbsp;
-			{ __( 'Advanced fraud protection', 'woocommerce-payments' ) }
-		</h2>
-		<p className="fraud-protection-advanced-settings-notice">
+		<h2>{ __( 'Filter configuration', 'woocommerce-payments' ) }</h2>
+		<p>
 			{ __(
-				'At least one risk filter needs to be enabled for advanced protection.',
+				'Set up advanced fraud filters. Enable at least one filter to activate advanced protection.',
 				'woocommerce-payments'
 			) }
 		</p>
 	</>
 );
 
-const SaveFraudProtectionSettingsButton: React.FC = ( { children } ) => {
-	const headerElement = document.querySelector(
-		'.woocommerce-layout__header-wrapper'
-	);
-	return headerElement && ReactDOM.createPortal( children, headerElement );
-};
+// Temporary solution until we have wider header redesign.
+const Breadcrumb = () => (
+	<>
+		<h2 className="fraud-protection-header-breadcrumb">
+			<small>
+				<Link
+					type="wp-admin"
+					href={ getAdminUrl( {
+						page: 'wc-settings',
+						tab: 'checkout',
+						section: 'woocommerce_payments',
+					} ) }
+				>
+					<span className="dashicons dashicons-arrow-left-alt2"></span>
```

Comment:
Does anyone has idea how it make to behave as here? https://github.com/Automattic/woocommerce-payments/pull/10634/files#diff-bcfd303d507c33a27671935e724ee64d0df30ff2a18bb6ef4c94f28d48190124R976-R994